modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/__snapshots__/DatePicker.es.js.snap

```diff
@@ -27,17 +27,18 @@ exports[`DatePicker has a predefinedValue 1`] = `
           <input
             name="datePicker"
             type="hidden"
-            value="06/02/2020"
+            value="Tue Jun 02 2020 00:00:00 GMT-0700 (Pacific Daylight Time)"
           />
           <input
             class="form-control input-group-inset input-group-inset-after"
             type="text"
-            value="06/02/2020"
+            value="Tue Jun 02 2020 00:00:00 GMT-0700 (Pacific Daylight Time)"
```

Comment:
@bryceosterhaus This is just a little strange, it may be that the DDM Date Picker implementation is changing the `dateFormat`, and probably with the entry of stabilizing `onValueChange` to return only `string` it may have broken something, I will check it today.

Reply:
Well, it doesn't seem to have affected the functionality but in some moments in the Form Builder, I can see this value in this format, then it goes back to the expected format, they have many implementations to deal with the internationalization of the Form Builder mechanism and with masks that left a lot complex that I’m not able to understand anymore, I tried to simplify some things but I ended up falling into more internationalization problems of the `moment` and the Form Builder, well I will not spend any more time on that. Maybe we can help them with mask simplification when we send https://github.com/liferay/clay/issues/4006